android/app/build.gradle

```diff
@@ -232,9 +232,9 @@ dependencies {
         implementation jscFlavor
     }
 
-    implementation 'software.aws.chimesdk:amazon-chime-sdk:0.15.0'
-    implementation 'software.aws.chimesdk:amazon-chime-sdk-media:0.15.0'
-    implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'software.aws.chimesdk:amazon-chime-sdk:0.17.9'
+    implementation 'software.aws.chimesdk:amazon-chime-sdk-media:0.17.10'
```

Comment:
The current Android media SDK has openssl version issue when submitting to Play Store, should we use a more stable version like 0.17.8, given that the version number in React demo will probably stay for a while before next update, and people may encounter this issue if they follow this.